# Takeoff behavior action definition

# Request
uint8 drone_id                                          # drone ID
float32 takeoff_height                                  # takeoff height (m)
---
# Result
bool takeoff_success                                    # false if failed to takeoff
---
# Feedback
geometry_msgs/TwistStamped actual_takeoff_speed         # actual speed (m/s)
float32 actual_takeoff_height                           # actual height (m)